README.md

```diff
@@ -38,7 +38,7 @@
   * Full Stack Frameworks
     * Meteor.js
     * Mean.js
-    * mern.io
+    * mern.js
```

Comment:
http://mern.io/

Reply:
Always verify your commits. `mern.js` doesn't even exist. 

Closing this PR for now.